plenty_api/api.py

```diff
@@ -1426,3 +1428,14 @@ def plenty_api_book_outgoing_items(self,
         logging.debug(response)
 
         return response
+
+    def plenty_api_update_property_selection_name(self, name_id: int, json: dict) -> dict:
```

Comment:
Please add a small function description, it doesn't need to be long, but there be a small description.